radar_view/RadarObj[] objects